service/src/test/java/org/sunbird/learner/actors/UserManagementActorTest.java

```diff
@@ -1,1770 +1,335 @@
-
-    Map<String, Object> request = new HashMap<String, Object>();
-    request.put(JsonKey.USER, innerMap);
-    reqObj.setRequest(request);
-
-    subject.tell(reqObj, probe.getRef());
-    ProjectCommonException response =
-        probe.expectMsgClass(duration("2000 second"), ProjectCommonException.class);
-    if (null != response) {
-      assertEquals(ResponseMessage.Message.INVALID_MEDIA_TYPE, response.getMessage());
-    }
-  }
-
-  @SuppressWarnings("deprecation")
-  @Test
-  public void testZFUpdateUserWithInValidWebPageURL() {
-    TestKit probe = new TestKit(system);
-    ActorRef subject = system.actorOf(props);
-
-    Request reqObj = new Request();
-    reqObj.setOperation(ActorOperations.UPDATE_USER.getValue());
-    Map<String, Object> innerMap = new HashMap<>();
-    innerMap.put(JsonKey.USERNAME, "sunbird_dummy_user_1919");
-    innerMap.put(JsonKey.EMAIL, "sunbird_dummy_user_1919@gmail.com");
-    innerMap.put(JsonKey.ID, userIdnew);
-    List<Map<String, String>> webPage = new ArrayList<>();
-    Map<String, String> webPageData = new HashMap<>();
-    webPageData.put(JsonKey.TYPE, "fb");
-    webPageData.put(JsonKey.URL, "https://test.com/test/");
-    webPage.add(webPageData);
-    innerMap.put(JsonKey.WEB_PAGES, webPage);
-
-    Map<String, Object> request = new HashMap<String, Object>();
-    request.put(JsonKey.USER, innerMap);
-    reqObj.setRequest(request);
-
-    subject.tell(reqObj, probe.getRef());
-    ProjectCommonException response =
-        probe.expectMsgClass(duration("2000 second"), ProjectCommonException.class);
-    if (null != response) {
-      assertEquals("Invalid URL for facebook", response.getMessage());
-    }
-  }
-
-  @Test
-  public void z10TestUpdateUserLoginTime() {
-    // setting emailVerified to true for testing
-    String respo = ssoManager.setEmailVerifiedTrue(userId);
-    System.out.println("respo " + respo);
-    try {
-      Thread.sleep(3000);
-    } catch (InterruptedException e) {
-      // TODO Auto-generated catch block
-      ProjectLogger.log(e.getMessage(), e);
-    }
-    TestKit probe = new TestKit(system);
-    ActorRef subject = system.actorOf(props);
-    Request reqObj = new Request();
-    reqObj.setOperation(ActorOperations.USER_CURRENT_LOGIN.getValue());
-    Map<String, Object> innerMap = new HashMap<>();
-    innerMap.put(JsonKey.USER_ID, userId);
-    Map<String, Object> request = new HashMap<String, Object>();
-    request.put(JsonKey.USER, innerMap);
-    reqObj.setRequest(request);
-    subject.tell(reqObj, probe.getRef());
-    Response response = probe.expectMsgClass(duration("200 second"), Response.class);
-    assertEquals(JsonKey.SUCCESS, (String) response.get(JsonKey.RESPONSE));
-  }
-
-  @Test
-  public void z11TestgetMediaTypes() {
-    TestKit probe = new TestKit(system);
-    ActorRef subject = system.actorOf(props);
-    Request reqObj = new Request();
-    reqObj.setOperation(ActorOperations.GET_MEDIA_TYPES.getValue());
-    subject.tell(reqObj, pr
```

Comment:
Function names are incorrect - like case-sensitivity issue. 
All the tests have same template - and code-duplicacy can easily be avoided.